hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/OzoneStreamCapabilities.java

```diff
@@ -44,4 +44,8 @@ private OzoneStreamCapabilities() {
    * TODO: If Hadoop dependency is upgraded, this string can be removed.
    */
   static final String UNBUFFER = "in:unbuffer";
+
+  static final String HSYNC = "hsync";
+
+  static final String HFLUSH = "hflush";
```

Comment:
Only if you update the patch anyway:

I think we can use these constants from Hadoop's `StreamCapabilities` directly.  `OzoneStreamCapabilities` was added when Ozone used Hadoop 3.2, which didn't have the other two constants.  (This reminds me to address the TODO above, filed HDDS-7863 for that.)